packages/flutter/test/cupertino/context_menu_test.dart

```diff
@@ -104,10 +103,7 @@ void main() {
       Rect decoyChildRect = tester.getRect(_findDecoyChild(child));
       expect(childRect, equals(decoyChildRect));
 
-      // TODO(justinmc): When ShaderMask is supported on web, remove this
-      // conditional and just check for ShaderMask.
-      // https://github.com/flutter/flutter/issues/52967.
-      expect(find.byType(ShaderMask), kIsWeb ? findsNothing : findsOneWidget);
+      expect(find.byType(ShaderMask), findsOneWidget);
```

Comment:
The test is getting skipped when `isBrowser` is true. Is the skipping still needed?